Makefile

```diff
@@ -464,13 +555,19 @@ $(BINDIR)/%/.:
 #
 # For all-test-code, we need to use -I test-code/<subdir>
 # Dependencies for the main executables
-COMPILE.c   = $(CC)  $(CFLAGS) $(INCLUDE) -c
-COMPILE.cpp = $(CXX) $(CFLAGS) $(INCLUDE) -c
-COMPILE.cc  = $(CXX) $(CFLAGS) $(INCLUDE) -c
+COMPILE.c       = $(CC)  $(CFLAGS) $(INCLUDE) -c
+COMPILE.cpp     = $(CXX) $(CFLAGS) $(INCLUDE) -c
+COMPILE.cc      = $(CXX) $(CFLAGS) $(INCLUDE) -c
+COMPILE.loc.c   = $(LOC_C_CC) $(CFLAGS) $(INCLUDE) -c
```

Comment:
This is really the new line: To specify that compiler will always be `gcc` when compiling `loc.c`, the LOC-source file where the main section-ID definition exists.